mmdet/models/detectors/yolact.py

```diff
@@ -30,7 +30,36 @@ def forward_dummy(self, img):
 
         See `mmdetection/tools/analysis_tools/get_flops.py`
         """
-        raise NotImplementedError
+        # img_metas do not influence the flops calculation
+        dummy_img_metas = [
+            dict(
+                img_shape=(300, 300),
```

Comment:
It might not be a good idea to hardcode the shape of images in this place. Can we use img.shape?